datawire-envoy-1.31.yaml

```diff
@@ -69,6 +70,16 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-setcap-ei
```

Comment:
Sorry to nitpick, but can we call this `-privileged` instead?

Reply:
Done.